altiProfilAdmin/controllers/config.classic.php

```diff
@@ -24,8 +24,12 @@ class configCtrl extends jController {
 
     function __construct( $request ) {
         parent::__construct( $request );
-        $monfichier = jApp::configPath('altiProfil.ini.php');
-        $this->ini = new jIniFileModifier ($monfichier);
+        if (method_exists('jApp', 'varConfigPath')) {
+		    $monfichier = \jApp::varConfigPath('altiProfil.ini.php');
+        } else {
+		    $monfichier = \jApp::configPath('altiProfil.ini.php');
+        }
+        $this->ini = new \Jelix\IniFile\IniModifier($monfichier);
```

Comment:
you should include also this change into the if.
`if (method_exists('jApp', 'varConfigPath'))  ->  \Jelix\IniFile\IniModifier else jIniFileModifier`